src/docs/transaction-clustering/index.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Sanitizing URL Transactions # TODO: decide on name.
+---
+
+Sentry attempts to scrub high-cardinality identifiers from URL transactions
+to give the user more meaningful transaction groups.
+
+In terms of user experience, this feature plays a similar role as [Issue Grouping](/grouping).
+In terms of technical implementation, it is similar to [Data Scrubbing](/pii).
+
+## The Problem
+
+In our [Performance](https://docs.sentry.io/product/performance/) product, transactions are grouped by their name
+(the [`event.transaction`](/sdk/event-payloads/#optional-attributes) field).
+This works well as long as the cardinality of distinct transaction names that the SDK sends is low, for example
+by using the [route of a web framework](https://docs.sentry.io/platforms/javascript/guides/react/configuration/integrations/react-router/) as transaction name.
+
+However, in some cases the SDK does not have enough information to pick a meaningful group
+like the name of a view or a route of a web transaction,
+and it has to fall back to the raw URL (or rather,
+[its path component](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#Syntax)).
+
+First and foremost, this makes it harder for the user to extract insights from [Performance
+metrics](https://docs.sentry.io/product/performance/metrics/), because instead of presenting averages, percentiles and distributions of groups of transactions that logically
+belong together, we end up with a bunch of one-off transaction groups.
+
+```
+/account/settings/nf215j634705291cvcm725757m53451365023487cv562n905n2634590xn26340/
+/issues/1234567890
+/saml/acs/my-organization/
+/settings/e1fb593b4c3a4d388e2f6ea8c92c3abc/
```

Comment:
These are from our javascript project, but I replaced the actual IDs by random keyboard entries.

Reply:
Although I think the examples are good, I think cooking one or two examples that are in line with the rest of the document is more helpful to understand what this is really about. Since we already have diagrams for `/user/alice` and `/blog`, I think it's a good chance to add `/blog/1234567dksajhfl/comments` (or similar) and have all cases covered.

Reply:
Good call, I aligned all the examples.